twtr/src/test/groovy/twtr/AccountSpec.groovy

```diff
@@ -105,26 +128,31 @@ class AccountSpec extends Specification {
         sus.errors["password"] != null
     }
 
-    void "Account password variations"(String thePassword, boolean isValid, boolean passwordError) {
-        expect:
+    @Unroll
+    void "Account password variations: #description"() {
```

Comment:
Did Mike mention that Unroll achieves the same goal as #description for data driven tests?